sos/report/plugins/sunbeam.py

```diff
@@ -37,10 +48,90 @@ def setup(self):
             'sunbeam cluster list --format yaml',
         ])
 
+        sunbeam_user = self.get_option("sunbeam-user")
+        user_pwd = pwd.getpwnam(sunbeam_user)
```

Comment:
If the user does not exist, `getpwnam` raises an exception that will end up in `sos_logs/sunbeam-plugin-errors.txt`:

    >>> pwd.getpwnam('no-user')
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    KeyError: "getpwnam(): name not found: 'no-user'"
    >>> 


Technically it doesnt matter much since even catching the exception will cause the plugin wont collect anything (due to `if user_pwd`), just that scenario looks bit non-professional.

Leaving on @arif-ali to decide if it is worth fixing it - I dont want to block this "form of error reporting" point from merging.

Reply:
good catch, I agree it's looks non-professional if that error file is created; I will fix and re-upload. Thanks for catching this